### DISCUSSION TITLE
Does Weather.gov/alerts and it's API has any pricing or it's free?

### DISCUSSION THREAD

**Discussion by developerToday** [weather-gov/api#424] [Q&A]
Please help me and suggest any reference links for pricing for weather.gov api's. TIA

**Comment by StephenClouse** [comment]
All of the information presented via the API is intended to be open data, free to use for any purpose.

On a legal level, all of the data from API is either:

- Work prepared by an officer or employee of the United States federal government as part of that person's official duties, and is thus in the public domain under US copyright law;
- Data that NOAA has explicit permission to redistribute publicly and without restriction; or
- Bare facts that are not copyrightable

As a public service of the United States Government, we do not charge any fees for the usage of this service, although there are reasonable rate limits in place to prevent abuse and help ensure that everyone has access.

**Reply by developerToday** [reply]
Cool. thanks for sharing. Do you have any public reference url documentation that i can go through.

**Reply by scadergit** [reply]
Both weather.gov/documentation and api.weather.gov/openapi.json

**Reply by developerToday** [reply]
@scadergit @StephenClouse thank you. I was asking public reference url which gives information about pricing :)

**Reply by scadergit** [reply]
I've added what @StephenClouse mentioned above to the documentation Overview tab.

**Reply by developerToday** [reply]
@scadergit yep got it. thank you!

**Reply by developerToday** [reply]
@StephenClouse @scadergit  I have gone through the Pricing section in weather.gov. May I know what is the meaning of `Proxies are more likely to reach the limit, whereas requests directly from clients are not likely.` from the below screenshot: 
<img width="1038" alt="Screen Shot 2021-07-07 at 8 55 52 AM" src="https://user-images.githubusercontent.com/68301254/124772364-94d6d280-df01-11eb-9378-d01950530c4c.png">
Looking forward for your reply. 

Do you have any estimation of how many requests we can do to weather.gov/alerts ?
